README.md

```diff
@@ -83,7 +83,7 @@ haproxy_exporter --haproxy.scrape-uri=unix:/run/haproxy/admin.sock
 To run the haproxy exporter as a Docker container, run:
 
 ```bash
-docker run -p 9101:9101 quay.io/prometheus/haproxy-exporter:v0.12.0 --haproxy.scrape-uri="http://user:pass@haproxy.example.com/haproxy?stats;csv"
+docker run -p 9101:9101 quay.io/prometheus/haproxy-exporter:v0.13.0 --haproxy.scrape-uri="http://user:pass@haproxy.example.com/haproxy?stats;csv"
```

Comment:
```suggestion
docker run -p 9101:9101 quay.io/prometheus/haproxy-exporter --haproxy.scrape-uri="http://user:pass@haproxy.example.com/haproxy?stats;csv"
```

Reply:
Yes makes sense. In this case suppose someone new is going to use it and don't know where to put tag, to help them I am going to mention quay.io/prometheus/haproxy-exporter:latest instead of leaving it empty